### ISSUE TITLE
Allow URI input form on replay to work without JS

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/ipwb#414]
Per #412, we added a button to submit the form, which previously relied on hitting enter with a text field in focus. The conventional submission mechanism is currently suppressed via JS and handled via JS.

This should be more accessible. @ibnesayeed provided a mechanism to accomplish this in #412.

**Comment by ibnesayeed** [comment]
Direct link to the related comment https://github.com/oduwsdl/ipwb/pull/412#issuecomment-402299144.

**Comment by machawk1** [comment]
Added a fix for this in b0108efa82695481293962b58ba6584e2a02d8a1 and tested without JS.

When `/memento/*/?url=foo.com` was requested, the normal handler `showMementosForURIRs(urir)` with route `/memento/*/<path:urir>` is not called due to the `path:` requirement specification. Instead, `show_uri(path, datetime=None)` would be called with the more generic route.

Creating a path-less route of simply `/memento/*/` then reading the query string and performing a redirect to `showMementosForURIRs(urir)` proved sufficient for reusing the function/route.
